src/components/NewsFeed.tsx

```diff
@@ -101,8 +103,40 @@ const NewsFeed = () => {
         console.error('[NEWSFEED ERROR]', err)
       })
 
+    const youtubeVideos = await youtubeApi
+      .get('')
+      .then((res) => {
+        const data = res.data.items
+
+        if (!data) {
+          return null
+        }
+
+        return data.map((article: YoutubeArticleProps) => {
+          return {
+            id: article.id,
+            title: article.snippet.title,
+            dateTime: article.snippet.publishedAt,
+            webLink: `https://www.youtube.com/watch?v=${article.snippet.resourceId.videoId}`,
+            mediaLink: article.snippet.thumbnails.medium.url,
+            description: removeMd(article.snippet.description)
+              .replace('\n', ' ')
+              .substring(0, 200),
```

Comment:
can you remove the `.substring(0, 200)` here? We are already limiting the number of lines up to 3 using chakra ui's prop. thanks!